components/fonts/shapers/harfbuzz.rs

```diff
@@ -218,22 +216,11 @@ impl Shaper {
     }
 }
 
-pub fn unicode_script_to_iso15924_tag(script: unicode_script::Script) -> u32 {
-    let bytes: [u8; 4] = match script {
-        unicode_script::Script::Unknown => *b"Zzzz",
-        _ => {
-            let short_name = script.short_name();
-            short_name.as_bytes().try_into().unwrap()
-        },
-    };
+impl THarfShaper for Shaper {
+    type ShapedGlyphData = ShapedGlyphData;
 
-    u32::from_be_bytes(bytes)
-}
-
-impl Shaper {
-    /// Calculate the layout metrics associated with the given text when painted in a specific
-    /// font.
-    pub(crate) fn shape_text(&self, text: &str, options: &ShapingOptions, glyphs: &mut GlyphStore) {
+    /// Calculate the layout metrics associated with the given text when painted in a specific font.
```

Comment:
Nit: Not only a specific font, but the font associated with this Shaper.
```suggestion
    /// Calculate the layout metrics associated with the given text when with the [`Shaper`]s font.
```